nucliadb_node/src/shards/shard_reader.rs

```diff
@@ -249,11 +249,12 @@ impl ShardReader {
     }
 
     #[tracing::instrument(skip_all)]
-    pub fn get_vectors_keys(&self) -> NodeResult<Vec<String>> {
-        read_rw_lock(&self.vector_readers)
-            .get(DEFAULT_VECTORS_INDEX_NAME)
-            .expect("Default vectors index should never be deleted (yet)")
-            .stored_ids()
+    pub fn get_vectors_keys(&self, vectorset_id: &str) -> NodeResult<Vec<String>> {
```

Comment:
this is only used for testing purposes, right?

Reply:
I think so, at least it wasn't implemented in the bindings nor properly implemented for vectorsets